src/pl/ob_pl_package_manager.cpp

```diff
@@ -173,7 +173,7 @@ int ObPLPackageManager::read_and_exec_package_sql(
                                                 static_cast<int64_t>(compa_mode)))) {
             LOG_WARN("fail to exec package sql", K(sql_buf), K(ret));
           } else if (affected_rows != 0) {
-            char *last_slash = strrchr(package_full_path, '/');
+            const char *last_slash = strrchr(package_full_path, '/');
```

Comment:
Don't introduce it in this PR for ease of merging.

Reply:
Got it, and it seems that I should combine the 2 commits into one final commit before merging because the changes are small. So I will exclude it when I do the combination.

PS: test is ongoing, I will make a new comment when I get the result  :）